Tests/Functional/Asynchronous/Expectations/main.swift

```diff
@@ -603,15 +633,20 @@ class ExpectationsTestCase: XCTestCase {
             ("test_expectationCreationOnSecondaryThread", test_expectationCreationOnSecondaryThread),
             ("test_expectationCreationWhileWaiting", test_expectationCreationWhileWaiting),
             ("test_runLoopInsideDispatch", test_runLoopInsideDispatch),
+
+            // waitForExpectations() + @MainActor
+            ("test_waitForExpectationsAsync", asyncTest(test_waitForExpectationsAsync)),
+            ("test_waitForExpectationsFromMainActor", asyncTest { test_waitForExpectationsFromMainActor($0) }),
+            ("test_waitForExpectationsFromMainActor_async", asyncTest { test_waitForExpectationsFromMainActor_async($0) }),
```

Comment:
Why can't this be
```suggestion
            ("test_waitForExpectationsFromMainActor_async", asyncTest(test_waitForExpectationsFromMainActor_async)),
```
Keep in mind that users never write `asyncTest(...)` or `asyncTest { ... }` themselves; SwiftPM inserts the former for them automatically if the test function has the `async` keyword

Reply:
I already answered that. https://github.com/apple/swift-corelibs-xctest/pull/452#discussion_r1242597379